src/com/pentalog/pentastagiu/homework2/PR3/Board.java

```diff
@@ -0,0 +1,208 @@
+                    while (appValidation != 1) {
+                        System.out.println("WRONG PASSWORD OR EMAIL! TYPE AGAIN");
+                        appValidation = login();
+                        if (appValidation == 99)
+                            break;
+                    }
+                    ////here- we will be entered in the app
+                    if (appValidation == 1) //we are in!! =D
+                    {
+                        ///another menu for the app
+                        ///1 - see all  messages
+                        ///2 - post new message
+                        ///3 - see only messages posted by yourself
+                        ///4 - log out and go back to main menu
+                        ///5 - exit the page totally
+                        System.out.println("Hi, there! You got in! Choose one of your options!\n1 - see all the messages ever posted\n" +
+                                "2 - post new message\n" +
+                                "3 - see only messages posted by yourself\n" +
+                                "4 - log out and go back to main menu\n" +
+                                "5 - exit the page totally ");
+                        int loginOption = input.nextInt();
+                        boolean altceva = true;
+                        while (altceva == true) {
+                            switch (loginOption) {
+                                case 1: {
+                                    seeAllMessages();
+                                    altceva = true;
+                                    System.out.println("Pick another option from the APP MENU! <3");
+                                    loginOption = input.nextInt();
+                                    break;
+                                }
+                                case 2: {
+                                    postNewMessage(email); ///this is gonna be a little harder
+                                    System.out.println("Your message was succesfully posted");
+                                    altceva=true;
+                                    System.out.println("Pick another option from the APP MENU! :*");
+                                    loginOption = input.nextInt();
+                                    break;
+                                }
+                                case 3: {
+                                    seeMyMessages(); /// i am gonna need the id- WHERE CURRENT USER email == message id- post it .  Current email0- last email typed
+                                    altceva = true;
+                                    System.out.println("Pick another option from the APP MENU!");
+                                    loginOption = input.nextInt();
+                                    break;
+                                }
+                                case 4: {
+
+                                    altceva = false; //this menu
```

Comment:
denumirile ar trebui sa fie in engleza si sugestive, de exemplu isOutterMenuOption si isInnerMenuOption